app/assets/stylesheets/custom.css

```diff
@@ -0,0 +1,114 @@
+/* General style */
+body {
+  background: #000 !important;
+  color: #eee !important;
+}
+
+a {
+  text-decoration: none !important;
+  color: #eee !important;
+}
+
+a:hover {
+  transform: translateY(4px);
+}
+
+a:active {
+  transform: translateY(1px);
+}
+
+.notice {
+  color: green !important;
+}
+
+.alert {
+  color: red !important;
+}
+
+.bg-orange {
+  background: rgb(241, 84, 17) !important;
+}
+
+/* label */
+.required::after {
+  content: "*";
+  color: red;
+}
+
+.btns {
+  background-color: rgb(52, 179, 241);
+  border-color: rgb(52, 179, 241);
+  border-radius: 20% 40%;
+  box-shadow: 0 0 4px #eee;
+  color: #fff;
+  font-weight: 500;
+}
+
+.logout-btn {
+  float: right;
+  margin-right: 10px;
+}
+
+input {
+  margin-bottom: 1em !important;
+}
+
+/* Splash page */
+.splash-page {
+  height: 50vh;
+  border-radius: 20% 40%;
+  background: rgb(241, 84, 17);
+  box-shadow: 0 0 6px #eee;
+}
+
+.splash-btns {
+  width: 100%;
+}
+
+.splash-btns:hover {
+  background-color: rgb(40, 167, 230);
+  transform: translateY(-2px);
+}
+
+/* Group page */
+.group-page {
+  height: 50vh;
+  border-radius: 20% 40%;
+  background: rgb(241, 84, 17);
+  box-shadow: 0 0 6px #eee;
+}
+
+.grids {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+  gap: 0.6em;
+  padding: 1em;
+  height: 200px;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.item {
+  padding: 1em;
+  border: 1px solid #eee;
+  border-radius: 20%;
+  background: rgb(241, 84, 17);
+  box-shadow: 0 0 6px #eee;
```

Comment:
* Kindly stick to design provided as it is a requirement 
* Note these includes Colour, typogragphy, font, weight, layouts, spacing etc [Please refer to the rpoject requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.)

Reply:
The style for small screens now fits with the design provided